www/nginx/src/opnsense/service/templates/OPNsense/Nginx/http.conf

```
@@ -247,6 +256,11 @@ server {
     }
 {%   endif %}
     include {{ server['@uuid'] }}_pre/*.conf;
+{%   if server.naxsi_whitelist_srcip is defined and server.naxsi_whitelist_srcip != '' %}
+     if ($naxsi_wlsrcip) {
+          set $naxsi_flag_enable 0;
```

Why not use the geo value here, then you don't need the if (needs flipping the 1/0 then)?
Do you plan to use other things for whitelisting?
```suggestion
          set $naxsi_flag_enable $naxsi_wlsrcip;
```

Have you checked if it will be enabled again on the next request - some variables keep their value?

---

@fabianfrz I've tried with your suggestion, but doesn't worked. 
Answering your question: By now, I've implemented it just to bypass the `SecRulesEnabled` directive and not block a trusted IP.

---

OK, then I am fine with it.